src/Atomic.Seo/Atomic.Sео.csproj

```diff
@@ -68,4 +68,8 @@
 		<ProjectReference Include="..\Atomic.Seo.ModelsBuilder\Atomic.Seo.ModelsBuilder.csproj" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <Folder Include="Html\Constants\" />
+	</ItemGroup>
+
```

Comment:
You could remove the ItemGroup for Html\Constants. We try to keep the csproj file as clean as possible. This one actually even adds broken ref in VS when I test.

Reply:
csproj is cleaned up